queries/scala/highlights.scm

```diff
@@ -0,0 +1,62 @@
+;; keywords
+
+[
+  "abstract"
+  "case"
+  "class"
+  "do"
+  "extends"
+  "final"
+  "finally"
+  "forSome"
+  "implicit"
+  "lazy"
+  "macro"
+  "new"
+  "null"
+  "object"
+  "override"
+  "package"
+  "private"
+  "protected"
+  "sealed"
+  "super"
+  "this"
+  "trait"
+  "type"
+  "val"
```

Comment:
Probably, one of theses keywords doesn't exist in the Tree-Sitter grammar. the CI complains about invalid nodes.

Reply:
done